app/javascript/chat/connectReducer.js

```diff
@@ -0,0 +1,307 @@
+export function connectReducer(state, action) {
```

Comment:
documentation (blocking): For any functions, constants etc. that are exported in an ES module, provide documentation using JSDoc comments. For an example, see https://github.com/forem/forem/blob/master/app/javascript/utilities/http/request.js#L1.